### PR TITLE
DDF-3579 edits to ingest docs (#2927)

### PR DESCRIPTION
**Pull request by ricklarsen**
**Port to `master` of PR https://github.com/codice/ddf/pull/2927**

#### What does this PR do?

Updates documentation for instructions on ingesting into DDF. 
- Reorganization
- Corrections
- New content

#### Who is reviewing it? 
@mcalcote @vinamartin @garrettfreibott @ahoffer 

#### Choose 2 committers to review/merge the PR. 
@clockard
@coyotesqrl
@lessarderic
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

Review content revisions. 

#### What are the relevant tickets?
[DDF-3579](https://codice.atlassian.net/browse/DDF-3579)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_running/directory-monitor-ingest.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+:parent: Ingesting Data
+:order: 02
+
+== {title}
+
+The ${ddf-catalog} application contains a Content Directory Monitor feature that allows files placed in a single directory to be monitored and ingested automatically.
+For more information about configuring a directory to be monitored, see <<_configuring_the_content_directory_monitor,Configuring the Content Directory Monitor>>.
+
+Files placed in the monitored directory will be ingested automatically.
+Ifa file cannot be ingested, they will be moved to an automatically-created directory named `.errors`.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_running/directory-monitor-ingest.adoc]: ✏️ need a space between `If` `a`

File: distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+The `--force` option removes without a confirmation message.
+
+The `--expired` option is to remove only expired records.
+
+[IMPORTANT]
+====
+If the `--expired` option is omitted, then all records will be removed from the Catalog.
+====
+
+The Catalog Command is convenient for removing records but it is not guaranteed to be fast or efficient. Additionally, certain configurations for archiving Catalog records can dramatically slow the process. If efficiency and speed are desired, interact directly with the catalog storage provider (e.g. Solr, Oracle) to delete records.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc]: ✏️ if you go through here again might be worth removing `Oracle` references

File: distribution/docs/src/main/resources/content/_running/directory-monitor-ingest.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_running/directory-monitor-ingest.adoc]: fixed

File: distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc]: dropped

**Comment by ricklarsen** [comment]
build now

File: catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java [hunk 3]
```diff
@@ -147,8 +147,8 @@
   @Argument(
     name = "File path or Directory path",
     description =
-        "File path to a record or a directory of files to be ingested. Paths are absolute and must be in quotes."
-            + " This command can only detect roughly 2 billion records in one folder. Individual operating system limits might also apply.",
+        "File path to a file or a directory of files to be ingested. Paths can be absolute or relative to installation directory."
```
Comment (mcalcote) [inline catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java]: ✏️ "Path to a file or ..."

File: catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java [hunk 4]
```diff
@@ -147,8 +147,8 @@
...
+            + " This command can only detect roughly 2 billion files in one folder. Individual operating system limits might also apply.",
```
Comment (mcalcote) [inline catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java]: ✏️ "...files in one directory. Individual..." 
`folder` should be reserved for gui references.

File: distribution/docs/src/main/resources/content/_running/auto-removing-expired.adoc [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+By default, the ${branding} runs a scheduled command every 24 hours to remove expired records.
+The command is executed and scheduled by the <<Command Scheduler>>.
+To change the configuration, follow the <<Updating a Scheduled Command>> instructions.
+If an administrator wants to create additional scheduled tasks to remove records from the local Catalog, the administrator can follow the steps provided in the Scheduling a Command section.
+In the Command text field, type the following:
+
+`catalog:removeall --force --expired`
+
+If it is intended to have this run daily, type 86400 for the number of seconds.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_running/auto-removing-expired.adoc]: ❓ wow, we really want to expire records on a per second basis 🤦‍♂️ ?

File: distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+When prompted, type `yes` to remove all expired records.
+
+[TIP]
+====
+For help on the `removeall` command, execute
+
+`help catalog:removeall`
+====
+
+The `catalog:removeall` command states you want to remove records from the local Catalog.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc]: ❓ I'm confused about this line... phrasing?

File: distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+The `--force` option removes without a confirmation message.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc]: ✏️ "... removes records without a ..."

File: distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc [hunk 8]
```diff
@@ -3,15 +3,25 @@
...
 :status: published
 :link: _tika_input_transformer
-:summary: Translates Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into Catalog Metacards.
+:summary: Translates Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into Catalog records.
 
-The Tika Input Transformer is the default input transformer responsible for translating Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into Catalog Metacards.
+The Tika Input Transformer is the default input transformer responsible for translating Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into Catalog records.
 This input transformer utilizes https://tika.apache.org[Apache Tika] to provide basic support for these mime types.
-As such, the metadata extracted from these types of documents is the metadata that is common across all of these document types, e.g., creation date, author, last modified date, etc.
+The metadata common to all of these document types, e.g., creation date, author, last modified date, etc., is extracted and used to create the catalog record.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc]: ✏️ "...to all these..."

File: distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc [hunk 9]
```diff
@@ -3,15 +3,25 @@
...
 The Tika Input Transformer's main purpose is to ingest these types of content into the Metadata Catalog.
 
-The Tika input transformer is given a service ranking (priority) of -1 so that it is guaranteed to be the last input transformer that is invoked.
-This allows any registered input transformer that are more specific for any of these document types to be invoked instead of this rudimentary input transformer.
+The Tika input transformer is most basic input transformer and the last to be invoked.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc]: 🔥

File: catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java (hunk 3 above)
Comment (ricklarsen) [reply catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java]: fixed.

File: catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java (hunk 4 above)
Comment (ricklarsen) [reply catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java]: fixed.

File: distribution/docs/src/main/resources/content/_running/auto-removing-expired.adoc (hunk 5 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_running/auto-removing-expired.adoc]: sure seems that way

File: distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc (hunk 6 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc]: rephrased

File: distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc (hunk 7 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc]: updated.

File: distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc (hunk 8 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc]: fixed

File: distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc (hunk 9 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_transformers/tika-input-xformer.adoc]: 🏀 🏀 🏀

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**